### ISSUE TITLE
RETURNING clause not working

### ISSUE THREAD

**Issue by dyozie** [greenplum-db/gpdb#4194]
The [PR 3252](https://github.com/greenplum-db/gpdb/pull/3252) merge introduces changes that show Greenplum supports the RETURNING clause, but it doesn't work in a recent build of master:

```
gpadmin=# \h insert
Command:     INSERT
Description: create new rows in a table
Syntax:
INSERT INTO table [ ( column [, ...] ) ]
    { DEFAULT VALUES | VALUES ( { expression | DEFAULT } [, ...] ) [, ...] | query }
    [ RETURNING * | output_expression [ [ AS ] output_name ] [, ...] ]

gpadmin=# \h delete
Command:     DELETE
Description: delete rows of a table
Syntax:
DELETE FROM [ ONLY ] table [ * ] [ [ AS ] alias ]
    [ USING usinglist ]
    [ WHERE condition | WHERE CURRENT OF cursor_name ]
    [ RETURNING * | output_expression [ [ AS ] output_name ] [, ...] ]

gpadmin=# INSERT INTO distributors (did, name) VALUES (DEFAULT, 'XYZ Widgets') RETURNING did;
ERROR:  The RETURNING clause of the INSERT statement is not supported in this version of Greenplum Database.
gpadmin=# DELETE FROM distributors WHERE name = 'fidgets' returning *;
ERROR:  The RETURNING clause of the DELETE statement is not supported in this version of Greenplum Database.
```

**Comment by Eulerizeit** [comment]
RETURNING is not working. The plan is to fix it during or after the Postgres 9.0 merge.

**Comment by Eulerizeit** [comment]
Fixed with https://github.com/greenplum-db/gpdb/pull/4531
